mods/gen1/formats-data.js

```diff
@@ -96,7 +97,7 @@ exports.BattleFormatsData = {
 		tier: "UU"
 	},
 	ekans: {
-		randomBattleMoves: ["wrap","glare","earthquake","bodyslam","rockslide"],
```

Comment:
Even if boring, Wrap is what defines Ekans, and actually should be mandatory move in my opinion.